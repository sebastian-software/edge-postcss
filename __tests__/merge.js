import { compile } from "./core"

// ====================================================
// ================== MERGING =========================
// ====================================================

test("Import Basic", () =>
  compile("@import './fixtures/import-a.css';")
)

test("Import with Merge", () =>
  compile("@import './fixtures/import-b.css'; .section { background: #333; }")
)
