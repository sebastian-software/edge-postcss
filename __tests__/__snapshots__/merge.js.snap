// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Import Basic 1`] = `
".root{
color:red
}"
`;

exports[`Import with Merge 1`] = `
".section{
color:#00d;
background:#333
}"
`;
